style.css

```diff
@@ -92,3 +93,265 @@ p {
 .icons li {
   list-style: none;
 }
+
+/* work and about me section */
+
+.mult-post {
+  grid-row: 1;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  border: 1px solid red;
+}
+
+.work {
+  display: grid;
+  display: block;
```

Comment:
I suggest you remove this property, `display:block`. Using it just renders your grid irrelevant. Instead, look for a way to make your grid work. Part of the project requirememnts is styling this section using `display: grid`. You can leave out the first element, since its positioning is different but for the rest of the cards, just use a container and style it using `display:grid` and instead of using `template-rows` mobile version uses columns